sys/sanitize.sh

```diff
@@ -3,6 +3,8 @@
 # SANITIZE="address signed-integer-overflow"  # Faster build
 SANITIZE=${SANITIZE:="address undefined signed-integer-overflow"}
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
this is an ugly bashism

Reply:
It's ugly, but it's the most cross platform solution in bash to reliably get the working directory of the script. I have yet to have it fail on me.